spec/example_app/app/dashboards/order_dashboard.rb

```diff
@@ -14,6 +14,8 @@ class OrderDashboard < Administrate::BaseDashboard
     line_items: Field::HasMany,
     total_price: Field::Number.with_options(prefix: "$", decimals: 2),
     shipped_at: Field::DateTime,
+    status: Field::Enum.with_options(
+      enum: { "pending" => 0, "confirmed" => 1 }),
```

Comment:
Looks like `Order.statuses` returns: `{"pending"=>0, "confirmed"=>1}`. Maybe we can use that to avoid the `.with_options` call.

Here's the reference: http://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html